drf_util/views.py

```diff
@@ -83,26 +121,46 @@ def get_serializer_by_action(self):
     def get_serializer_class(self):
         return self.get_serializer_by_action() or super().get_serializer_class()
 
+    @staticmethod
+    def make_permissions(classes: list = None):
+        permissions = []
+
+        if classes is None:
+            classes = []
+
+        if not isinstance(classes, list) and issubclass(type(classes), BasePermission):
+            classes = [classes]
```

Comment:
I think the first if is extra, because list is not BasePerimission instance
You can try:
```
if not hasattr(classes, '__iter__'):
     classes = [classes]
```